### PR TITLE
Let plt.rc = matplotlib.rc, instead of being a trivial wrapper.

### PR DESCRIPTION
**Pull request by anntzer**
We previously had, in pyplot.py

    def rc(...): matplotlib.rc(...)

plus some docstring copying.

Instead, one can just do

    rc = matplotlib.rc

and likewise for a few other functions.

(Doing ``from matplotlib import rc``, which may seem even simpler, would
cause linters to complain about unused names.)

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/pyplot.py [hunk 1]
```diff
@@ -333,10 +324,9 @@ def gci():
 ## Any Artist ##
 
 
-# (getp is simply imported)
-@docstring.copy(_setp)
-def setp(obj, *args, **kwargs):
-    return _setp(obj, *args, **kwargs)
+get = matplotlib.artist.get
```
Comment (jklymak) [inline lib/matplotlib/pyplot.py]: Can we add a short comment like the one above.  These don't look like API to me when I look at them, they look like local aliases to the methods that someone didn't import right...

File: lib/matplotlib/pyplot.py [hunk 2]
```diff
@@ -1997,14 +1987,8 @@ def set_cmap(cmap):
...
 
-@docstring.copy_dedent(matplotlib.image.imread)
-def imread(fname, format=None):
-    return matplotlib.image.imread(fname, format)
-
-
-@docstring.copy_dedent(matplotlib.image.imsave)
-def imsave(fname, arr, **kwargs):
-    return matplotlib.image.imsave(fname, arr, **kwargs)
+imread = matplotlib.image.imread
```
Comment (jklymak) [inline lib/matplotlib/pyplot.py]: ditto

**Comment by tacaswell** [comment]
Good catch @timhoffm !

**Comment by anntzer** [comment]
Good catch.
While it may be possible to fix this on autosummary's side, I don't have the time to check autosummary's implementation details for now...

**Pull request closed without merging**